cli/src/cli.rs

```diff
@@ -151,6 +151,11 @@ pub struct RunCmd {
 pub struct Cli {
 	#[command(subcommand)]
 	pub subcommand: Option<Subcommand>,
+
 	#[clap(flatten)]
 	pub run: RunCmd,
+
+	#[allow(missing_docs)]
```

Comment:
```suggestion
```
The `allow(missing_docs)` directive on the struct should cover its members as well

Reply:
https://github.com/paritytech/polkadot/pull/6619/commits/4cea6fed2ee475a65db81d0726a5fdd6ba5ed3e7